tests/fast-integration/kyma-environment-broker/client.js

```diff
@@ -153,6 +157,14 @@ class KEBClient {
       return new Error(`error while deprovisioning SKR: ${err.toString()}`);
     }
   }
+
+  getRegion() {
+    let reg = this.region
```

Comment:
It may happen that `this.region` will be undefined (when the env variable is not present.) The condition in the if statement will return true, which I think is wrong. The concatenation will produce `undefined/`. The quickest fix for that case would be:
```
if (reg && reg != "")
```